docs/university/core-components/README.md

```diff
@@ -1,39 +1,192 @@
 # 🧩 Core Components
 
+<!-- ## GENERAL COMPONENTS -->
+<!-- Needs content
```

Comment:
why are these blocks commented out? the idea for creating these placeholder pages was that so people can contribute content easier, without the need for discussing the creation of the page